mkosi/distributions/opensuse.py

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
+import urllib.request
+import xml.etree.ElementTree as ET
```

Comment:
`import xml.etree.ElementTree as ElementTree` ? It is used exactly once, so this is just obfuscation.